README.md

```diff
@@ -21,7 +21,7 @@
 Install the components that you need from npm:
 
 ```sh
-npm install @vaadin/vaadin-grid
+npm install @vaadin/grid
```

Comment:
Let's revert changes in README for now. The stable 21.0.2 still uses `@vaadin/vaadin-grid` package.